packages/core/src/customer/customer-request-options.ts

```diff
@@ -31,6 +31,8 @@ export interface CustomerRequestOptions extends RequestOptions {
  * information in order to initialize the customer step of checkout.
  */
 export interface CustomerInitializeOptions extends CustomerRequestOptions {
+    [key: string]: unknown;
```

Comment:
I don't think you need this.

Reply:
I rebased and this was still needed for the types to match :|